docs/source/checks/nlp/model_evaluation/plot_prediction_drift.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__prediction_drift:
+
+Prediction Drift
+***************************
+
+This notebook provides an overview for using and understanding the NLP prediction drift check.
+
+**Structure:**
+
+* `What Is Prediction Drift? <#what-is-prediction-drift>`__
+* `Get Data and Predictions <#get-data-and-predictions>`__
+* `Run Check <#run-check>`__
+
+What Is Prediction Drift?
+=========================
+
+Drift is simply a change in the distribution of data over time, and it is
+also one of the top reasons why machine learning model's performance degrades
+over time.
+
+Prediction drift is when drift occurs in the prediction itself.
+Calculating prediction drift is especially useful in cases
+in which labels are not available for the test dataset, and so a drift in the predictions
+is our only indication that a changed has happened in the data that actually affects model
+predictions. If labels are available, it's also recommended to run the
+:doc:`Label Drift check </checks_gallery/nlp/train_test_validation/plot_label_drift>`.
```

Comment:
This link is no longer correct in the new branch. Should be 
```suggestion
:doc:`Label Drift check </nlp/auto_checks/train_test_validation/plot_label_drift>`.
```